ownerdelete.user.js

```diff
@@ -163,6 +170,22 @@ dry.once("load", () => {
     }
     return file;
   };
+  async function getInfo(file) {
+    try {
+      if (file.id in checksums && !file.checksum) {
+        file.checksum = checksums[file.id];
+        return;
+      }
+      const info = await dry.exts.info.getFileInfo(file.id);
+      const {checksum} = info;
+      checksums[file.id] = checksum;
+      file.checksum = checksum;
+      save_checksums();
```

Comment:
This shouldn't be in the hot path. 1000 files -> 1000 session storage saves is excessive.
E.g. you could start a timer, if there isn't one yet, to save once after a certain amount of time to effectively coalesce saves.